plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -422,6 +422,15 @@ public synchronized Optional<ConnectorOutputMetadata> finishCreateTable(
         TableInfo info = tables.get(tableName);
         requireNonNull(info, "info is null");
 
+        if (!computedStatistics.isEmpty() && !info.properties().containsKey(TableInfo.DEFAULT_LIMIT_PROPERTY)) {
+            ComputedStatistics statistic = computedStatistics.stream().findFirst().orElseThrow();
+            long rowCount = (long) firstNonNull(readNativeValue(BIGINT, statistic.getTableStatistics().get(ROW_COUNT), 0), 0);
```

Comment:
why 0 as default?

Reply:
I moved the default to where values are read from the map to make it more obvious.